internal/broker/client.go

```diff
@@ -160,6 +178,35 @@ func (c *Client) GetInstanceRequest(instanceID string) (response *http.Response,
 	return resp, nil
 }
 
+// Unbind requests Service Binding unbinding in KEB with given details
+func (c *Client) Unbind(binding internal.Binding) error {
+	unbindURL, err := c.formatUnbindUrl(binding)
+	if err != nil {
+		return err
+	}
+
+	log.Infof("sending unbind request for service binding with ID %q and instance ID: %q", binding.ID, binding.InstanceID)
+
+	emptyResponse := &apiresponses.EmptyResponse{}
+	for requestAttemptNum := 1; requestAttemptNum <= c.RequestRetries; requestAttemptNum++ {
+		if err = c.executeRequest(http.MethodDelete, unbindURL, http.StatusOK, nil, emptyResponse); err != nil {
```

Comment:
How do we handle 410 response? The `Unbind` endpoint will return 410 if Service Instance had been removed before DELETE was called.

Reply:
atm `executeRequest` method will throw an error and the job quits. Let me improve the method to return an error with HTTP status code from the response, so the job could log warning and act accordingly, that is continue its work when the instance or binding are gone.

Reply:
Discussed in direct communication.